### PR TITLE
Feature/rm infile lane prefix

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR adds | fixes:

- Removes the use infile_lane_prefix from MIP
- This information is now entirely keept in file_info instead

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [x] Installation, unit and integration test suite pass

### Review:
- [x] Code review
- [x] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: lib/MIP/Processmanagement/Processes.pm [hunk 1]
```diff
@@ -146,20 +146,20 @@ sub add_sample_ids_job_ids_to_job_id_dependency_string {
...
-## Arguments: $case_id                 => Case id
-##          : $case_id_chain_key       => Case id chain hash key
-##          : $infile_lane_prefix_href => The infile(s) without the ".ending" {REF}
-##          : $job_id_href             => The info on job ids hash {REF}
-##          : $path                    => Trunk or branch
-##          : $sample_ids_ref          => Sample ids {REF}
+## Arguments: $case_id                           => Case id
+##          : $case_id_chain_key                 => Case id chain hash key
+##          : $job_id_href                       => The info on job ids hash {REF}
+##          : $max_parallel_processes_count_href => Maximum number of parallel processes
```
Comment (jemten) [inline lib/MIP/Processmanagement/Processes.pm:152]:
```suggestion
##          : $max_parallel_processes_count_href => Maximum number of parallel processes {REF}
```

File: lib/MIP/Processmanagement/Processes.pm [hunk 2]
```diff
@@ -440,21 +443,21 @@ sub add_job_id_dependency_tree {
...
-##          : $job_id_href             => Info on jobIds hash {REF}
-##          : $path                    => Trunk or branch
-##          : $sample_id               => Sample ID
-##          : $sample_id_chain_key     => Sample ID chain hash key
+## Arguments: $case_id_chain_key                 => Case id chain hash key
+##          : $job_id_href                       => Info on job_ids hash {REF}
+##          : $max_parallel_processes_count_href => Maximum number of parallel processes
+##          : $path                              => Trunk or branch
+##          : $sample_id                         => Sample ID
+##          : $sample_id_chain_key                => Sample ID chain hash key
```
Comment (jemten) [inline lib/MIP/Processmanagement/Processes.pm:453]:
```suggestion
##          : $sample_id_chain_key               => Sample ID chain hash key
```

File: lib/MIP/Processmanagement/Processes.pm (hunk 1 above)
Comment (jemten) [reply lib/MIP/Processmanagement/Processes.pm:152]: And all other `##          : $max_parallel_processes_count_href => Maximum number of parallel processes` in this file

**Comment by henrikstranneheim** [comment]
Nice Thx! I will wait for travis and then merge

**Pull request merged**